.github/workflows/submodule_update.yml

```diff
@@ -4,50 +4,71 @@ on:
   push:
     branches:
       - 'master'
-      - '2.10'
+      - '[2-9].\d+'
 
 jobs:
   tarantool-ee:
     # Run on push to the 'master' and release branches in tarantool/tarantool
     # repo only.
     if: github.repository == 'tarantool/tarantool'
 
-    runs-on: ubuntu-20.04-self-hosted
+    runs-on: [self-hosted, Linux, x86_64, flavor-1-2]
```

Comment:
Could you please add spaces after `[` and before `]`?

Reply:
fixed